Canvas/Canvas/CanvasAppDelegate+Helm.swift

```diff
@@ -64,7 +64,11 @@ extension AppDelegate: RCTBridgeDelegate {
         // Pages
         let pageParamsToProps = { (parameters: [String: Any]) -> [String: Any]? in
             guard let courseID = try? parameters.stringID("courseID") else { return nil }
-            guard let url: String = try? parameters <| "url" else { return nil }
+            var parsedUrl: String? = try? parameters <| "url"
+            if parsedUrl == nil {
+                parsedUrl = try? parameters.stringID("url")
+            }
+            guard let url = parsedUrl else { return nil }
```

Comment:
could we just use `stringID` initially, like `courseID` does above? Or would that mess up some other cases?